frontend/views/components/CreateGroupSteps/GroupMincome.vue

```diff
@@ -10,12 +10,12 @@
         ref='mincome'
         type='number'
         placeholder='Amount'
-        name='incomeProvided'
+        name='mincomeAmount'
         step='1'
         min='0'
         required=''
-        :class='{ error: v.incomeProvided.$error }'
-        :value='group.incomeProvided'
+        :class='{ error: v.mincomeAmount.$error }'
+        :value='group.mincomeAmount'
```

Comment:
Noticing two things here:

1. Here we have the `v` thing going on again (#661). Is this something you think you can look into for this PR?
2. I ask that because in other parts of this PR you also noticed that the vuelidate instructions tell you to bind `$model` to `v-model`, through the `$v` (not `v`..) injected instance. And here we have instead `:value` being bound to `group.mincomeAmount`, which I think is a mistake (?) given that the Vue.js docs say explicitly that `input` form elements need to be bound using `v-model`

Reply:
Actually doing this I finally understood the difference.
`v` here is a prop passed at `CreateGroup.vue` as `:v='$v.form'`. That's why it has a prop validation `v: { type: Object }`.
2. Vuelidate docs say explicitly one way of binding a value to an input, but it can be done in other ways. In this particular case, we don't assign it directly because the values are passed by a prop `group` from the parent (CreateGroup.vue)  and then we pass the new value back to the parent through the `@input` event. It's all fine here. :)